src/cobra/test/test_core/test_dictlist.py

```diff
@@ -9,7 +9,6 @@
 from pickle import HIGHEST_PROTOCOL, dumps, loads
 
 import pytest
-from six.moves import range
```

Comment:
Was `range` an unused import? It's not removed from any statement in the module.

Reply:
I think it shadowed Python 3's `range`.